docs/release-1.2/existing-rhdh.md

```diff
@@ -64,121 +91,156 @@ metadata:
 EOF
 ```
 The value of `.data.npmrc` points to https://npm.registry.redhat.com.
-For testing RC plugin versions, update to `cmVnaXN0cnk9aHR0cHM6Ly9ucG0uc3RhZ2UucmVnaXN0cnkucmVkaGF0LmNvbQo=` (points to https://npm.stage.registry.redhat.com and can be accessed internally). If there is a need to point to multiple registries, modify the content of the secret's data from:
-```
+For testing RC plugin versions, update to `cmVnaXN0cnk9aHR0cHM6Ly9ucG0uc3RhZ2UucmVnaXN0cnkucmVkaGF0LmNvbQo=` (which points to https://npm.stage.registry.redhat.com and can be accessed internally within Red Hat). If there is a need to point to multiple registries, modify the content of the secret's data from:
```

Comment:
`can be accessed internally within Red Hat`
it brings clarity and I like this change.
I have a doubt about the sentence: this doc is for customers outside our realm. Can we clarify what an external user can do?

Reply:
I had the same feeling about it. How about something like:

> If including plugins from a different NPM registry, the `.data.npmrc` value should be updated with the base64 encoded NPM registry. Example: The Red Hat internal stage NPM registry is `https://npm.stage.registry.redhat.com`, so the value would be updated to `cmVnaXN0cnk9aHR0cHM6Ly9ucG0uc3RhZ2UucmVnaXN0cnkucmVkaGF0LmNvbQo=`  (Note: The Red Hat internal stage registry is only accessible internally within Red Hat)

And also maybe we should just drop the RH stage example and use `https://registry.npmjs.org/` as that would apply more to external users. 

Wdyt?

Reply:
100% agree and this looks clear.
The staging one should be mentioned internally in our docs, but not on this github.

Reply:
> drop the RH stage example and use https://registry.npmjs.org/
Agree, @batzionb can we do that?